### ISSUE TITLE
Filtering with DescriptorRef

### ISSUE THREAD

**Issue by chhex** [fabric8io/docker-maven-plugin#542]
First-off best thanks for the really great D-M-P ... tremendous....
Second i don't know if this is really an issue: i simply havn't found a reference in the documentation. 

The following works great: 
`    <assembly>
                 .....
                      <descriptorRef>some-assembly-ref</descriptorRef>
                ......
      </assembly>
`
With Maven one can have "filters" , some thing like the following: 

`
   <filters>
            <filter>src/main/assembly/myfilter.properties</filter>
    </filters>
`
Or is there another way to support filtering in assembly descriptors? 

Best thanks

**Comment by rhuss** [comment]
Unfortunately this is currently not possible directly, but you could use a construct with the `maven-resource-plugin` as described in https://github.com/fabric8io/docker-maven-plugin/issues/330#issuecomment-173608906 for filtering an external Dockerfile. But you are right, this would be a good addition.

**Comment by chhex** [comment]
Best thanks! 

Theoretically Maven Assembly supports a filtered: Whether to filter symbols in the files as they are copied, using properties from the build configuration. 

But unfortunately that does'nt work or is'nt true if you use your own ContainerDecriptorHandler for your Assembly.  In the end that's where i put the filtering logic.  

So the requirement is for me really nice to have ;-) ..... 

Keep on with the great work, thanx!

**Comment by rhuss** [comment]
Yes, shouldn't be hard to add. I'll add it on the list for one of the next releases.
